### PR TITLE
Inventory updates

### PR DESCRIPTION
**Pull request by jordangov**
A few projects... including Code.mil itself. :)

### CODE REVIEW

File: code.json [hunk 1]
```diff
@@ -135,6 +135,90 @@
         "created": "2017-12-18",
         "lastModified": "2018-02-13"
       }
-    }
+    },
+		{
```
Comment (jgarber623-gov) [inline code.json]: Hard to tell from the GitHub UI, but the indentation of either line 138 or 139 seems slightly off.

File: code.json [hunk 2]
```diff
@@ -135,6 +135,90 @@
...
+			},
+			"laborHours": 224,
+			"date": {
+				"created": "2016-10-01",
+				"metadataLastUpdated": "2018-02-13T13:00-04:00"
+			}
+		},
+		{
+			"organization": "Air Force",
+			"name": "Salvaging Static Analysis (SalSA)",
```
Comment (jgarber623-gov) [inline code.json]: :joy:

File: code.json (hunk 1 above)
Comment (jordangov) [reply code.json]: Damn tabs vs spaces.

**Comment by jordangov** [comment]
@jgarber623-gov Can I get a review sign off now? Any other issues?

**Pull request merged**